_projects/area_indicators.md

```diff
@@ -0,0 +1,60 @@
+---
+published: false # change to true once ready to post
+
+title: "Climate Change API Area Indicators" # project title inside quotes
+excerpt: "A high performance Scala API for calculating climate indicators over geospatial areas" # shows on project list page
+
+# project attributes
+requirements: # bullet list of requirements – one requirement per line – follow below format
+  - "Scala programming experience"
+  - "Python programming experience not required but preferred"
+  - "NetCDF experience not required but preferred"
+tags: # one tag per line – spaces are allowed in tags – follow below format
+  - "scala"
+  - "climate change"
+difficulty: "Hard" # easy, medium, hard – pick one
+mentors: # github username without the @ – example: designmatty
+  - "maurizi"
```

Comment:
Is there a way to note that I'd only be available as a secondary mentor? I'd like to be involved in project planning if someone takes this project on, but I don't have the time capacity to do day-to-day code review / pairing.

Reply:
I am prepared and happy to participate, but as I mentioned to Mike in another forum, I may have other fellowship responsibilities.  I do not know what the rules are concerning this.